docs/wiki/development/windows-provisioning.md

```diff
@@ -20,6 +20,17 @@ The PowerShell script `provision.ps1` is used to prepare a clean Windows 10 64 b
  * Generate the Visual Studio 2015 solution files: `cmake ..\.. -G "Visual Studio 14 2015 Win64"`
  * There should be a `OSQUERY.sln` in the build folder. Open this with Visual Studio 2015 that is already installed via the provisioning script.
 
+## Building `osqueryd.exe` and `osqueryi.exe`
```

Comment:
#2293 adds a script that should do a large amount of this for you, and after running the `tools/make-win64-dev-env.bat` and then `tools/make-win64-binaries.bat` one should have `osqueryi.exe` and `osqueryd.exe`.  Would you mind mentioning that in here?